api/package.json

```diff
@@ -15,6 +15,11 @@
     "seed": "knex seed:run --specific=main.js",
     "seed-dev": "NODE_ENV=development DEV_DB_HOST=localhost npm run seed",
     "seed-users": "knex seed:run --specific=base-users.js",
+    "make-mongoose-migrate": "migrate --config mongo-migrate.js create",
+    "make-mongoose-migrate-dev": "MONGO_URL=mongodb://mongo:cms@localhost:27017/eapd npm run make-mongoose-migrate",
+    "mongoose-migrate": "migrate --config mongo-migrate.js up",
+    "mongoose-migrate-dev": "MONGO_URL=mongodb://mongo:cms@localhost:27017/eapd npm run mongoose-migrate",
+    "mongoose-migrate-down": "migrate --config mongo-migrate.js down",
```

Comment:
The -dev version of this is missing.  However, I think the -dev versions are unnecessary.  How do you feel about updating the readme to indicate that someone should run ```export MONGO_URL=mongodb://mongo:cms@localhost:27017/eapd
``` before using these commands.  In addition is seems that for the create and down commands I need to add -- <NAME> to the command which wasn't super clear  maybe that can go in the readme too?

Reply:
yeah that's a good idea, I'll update that and then move it back to code review